app/src/PlayersList.js

```diff
@@ -24,17 +25,25 @@ export default class PlayersList extends React.Component {
     }
   }
 
+  async followPlayer(playerTo) {
+    const followingList = await this.app.genericApi1('cl_following', playerTo);
```

Comment:
Would probably be nice to have some error handling here. I see that genericApi1 is returning a promise, but i don't see you supplying a resolve/reject callback to the promise. (I understand that there might be reasons why you find it unnecessary to handle that; my comment is just for general cases)

Reply:
`this.app.genericApi1` has a default error handler defined in `App` class.
So, we do not need to handle the error again here.